ckan/requirements.txt

```diff
@@ -60,11 +60,11 @@ mako==1.1.4
 markdown==3.1.1
 markupsafe==1.1.1
 messytables==0.15.2
-newrelic==5.24.0.153
+newrelic==6.0.1.155
```

Comment:
Maybe you should limit the new relic version in the toml file to avoid this update